localstack/services/opensearch/cluster_manager.py

```diff
@@ -165,14 +168,11 @@ def create(
         # determine engine type
         engine_type = versions.get_engine_type(version)
 
-        # build final endpoint and cluster url
-        endpoint = build_cluster_endpoint(
+        url = build_cluster_endpoint(
             DomainKey.from_arn(arn), custom_endpoint, engine_type, preferred_port
         )
```

Comment:
I reenabled this after we talked about removing it @alexrashed . Passing down the information/extracting it in bits and pieces to avoid reverse parsing the urls parts is probably harder to read, it might be better to keep it that way so you do not have to worry about the server internals